python/ray/experimental/named_actors.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ray
+import ray.cloudpickle as pickle
+"""
+This file contains functions intended to implement the named actor
+"""
+
+def _calculate_key_(name):
+    return b"Actor:" + str.encode(name)
+
+def get_actor(name):
```

Comment:
Please add docstrings for these methods with the format used e.g., in

https://github.com/ray-project/ray/blob/4584193308a749bdea6dd757fc0148c2950d0b30/python/ray/worker.py#L2443-L2458